### ISSUE TITLE
Magazine page subscription form

### ISSUE THREAD

**Issue by frnsys** [misfist/tni-theme#10]
The magazine page subscription form should work similarly to the single post subscription form (#9).

e.g. `/magazine/sciencefiction/`:

![image](https://cloud.githubusercontent.com/assets/1059947/24257884/6c1d55c0-0fc2-11e7-870a-6f444526d00e.png)

The `Manage Your Subscription` link should go to `SUBSCRIPTION_HOST/manage` where `SUBSCRIPTION_HOST=https://subscriptions.staging.darkinquiry.com` for now.

**Comment by misfist** [comment]
Can that just be a regular link? If so, it can be added to widget area. 

`/wp/wp-admin/widgets.php`

**Comment by frnsys** [comment]
Yup a regular link will work for the `Manage Your Subscription` link, I'll make that change.
